buildenv/jenkins/docker-slaves/README.md

```diff
@@ -0,0 +1,83 @@
+<!--
+Copyright (c) 2018, 2018 IBM Corp. and others
+ 
+ This program and the accompanying materials are made available under
+ the terms of the Eclipse Public License 2.0 which accompanies this
+ distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ or the Apache License, Version 2.0 which accompanies this distribution 
+and
+ is available at https://www.apache.org/licenses/LICENSE-2.0.
+ 
+ This Source Code may also be made available under the following
+ Secondary Licenses when the conditions for such availability set
+ forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ General Public License, version 2 with the GNU Classpath
+ Exception [1] and GNU General Public License, version 2 with the
+ OpenJDK Assembly Exception [2].
+ 
+ [1] https://www.gnu.org/software/classpath/license.html
+ [2] http://openjdk.java.net/legal/assembly-exception.html
+ 
+ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH 
+Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+# Docker
+## About
+Dockerfiles are used to define how a docker container/image
+should be built. Docker containers are essentially virtual
+machines designed for short term use. This makes them very
+useful for testing purposes as you can generate a new container
+for every test and you are guaranteed to run against the
+same environment every time.
+
+## Status of Dockerfiles on different Platforms
+|           |     x86      |     s390x     |    ppc64le    |
+|:---------:|:------------:|:-------------:|:-------------:|
+| Ubuntu16  |  Test Only   | Build & Test  |   Test Only   |
+| Ubuntu18  |  Test Only   |   Test Only   |   Test Only   |
+| Centos6.9 | Build & Test | Not Available | Not Available |
+| Centos7   | To be Added  |  To be Added  | Build & Test  |
+| Windows   | To be Added  | Not Available | Not Available |
```

Comment:
May be better to say Not Applicable instead of Not Available (in cases where it will never be applicable).

Reply:
Or maybe just N/A?